core/CliMulti/Process.php

```diff
@@ -236,4 +240,18 @@ private static function isProcFSMounted()
         return strpos($type, 'proc') === 0;
     }
 
+    /**
+     * @return int[] The ids of the currently running processes
+     */
+     static function getRunningProcesses()
+    {
+        $ids = explode("\n", trim(`ps ex 2>/dev/null | awk '{print $1}'`));
+
+        $ids = array_map('intval', $ids);
+        $ids = array_filter($ids, function ($id) {
+            return $id > 0;
+        });
+
+        return $ids;
+    }
```

Comment:
Not required for this PR, but do you think it would be useful to have the check done above as a diagnostic? We could display process ids found by the command, maybe it would help users debug or reduce support requests on our end...